org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessModalDialog.java

```diff
@@ -68,6 +69,28 @@ public ProcessModalDialog(int WindowNo, ProcessInfo pi, boolean autoStart)
 		this(null, WindowNo, pi, autoStart);
 	}
 	
+	/**
+	 * @param WindowNo
+	 * @param pi
+	 * @param autoStart
+	 */
+	public ProcessModalDialog(EventListener<Event> listener, int WindowNo, ProcessInfo pi, boolean autoStart, boolean loadSavedParams)
```

Comment:
it seems this is always call with autoStart = false and loadSavedParams = true, if that's the case, simpler and more readable to just remove the 2 process parameter and use the false and true directly.